project/src/main/java/com/google/sps/data/Productset.java

```diff
@@ -0,0 +1,12 @@
+package com.google.sps.data;
+
+
+public class Productset{
```

Comment:
Java variable names should be camel case, so this class needs to be named ProductSet, not Productset and you'll need to update the references in the rest of the code.